src/components/content-tab-assistant.tsx

```diff
@@ -345,11 +345,13 @@ export function ContentTabAssistant( { selectedSite }: ContentTabAssistantProps
 		chatSelectors.selectIsLoading( state, instanceId )
 	);
 	const { userCanSendMessage } = usePromptUsage();
-	const { messages: welcomeMessages, examplePrompts } = useWelcomeMessages();
 	const isOffline = useOffline();
 	const { __ } = useI18n();
 	const lastMessage = messages.length === 0 ? undefined : messages[ messages.length - 1 ];
 	const hasFailedMessage = messages.some( ( msg ) => msg.failedMessage );
+	const { data, isLoading } = useGetWelcomeMessagesQuery( undefined, {
+		skip: ! client || isOffline,
```

Comment:
What would happen if we didn't disable the query when `client` is undefined? 🤔 Having to always pass the `skip` option for this purpose seems slightly tedious.